docs/rules/array-type/index.html

```diff
@@ -15,9 +15,9 @@
     - generic
     - array-simple
 optionExamples:
-  - '[true, array]'
-  - '[true, generic]'
-  - '[true, array-simple]'
+  - '[true, "array"]'
```

Comment:
these docs are autogenerated from the [source code](https://github.com/palantir/tslint/blob/2218cfc63b8282b9082a89ce03b395c3d9daa4cd/src/rules/arrayTypeRule.ts#L24), you'll need to make the changes there.